packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -205,85 +204,6 @@ void main() {
     );
   });
 
-  group('doctor usage params', () {
```

Comment:
These scenarios are covered later on in the file: 

https://github.com/flutter/flutter/blob/c3b92f7f208ef94afc8bca6647fe19c00ea3c686/packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart#L812-L813